Frontend/Shared/Rating.razor

```diff
@@ -0,0 +1,8 @@
+﻿<i class="fa stars rating-color">
+	<i class="fa stars stars-child rating-uncolored" style="width: @(100 - ReviewAverage * 100 / 5)%"></i>
```

Comment:
It looks like this can only handle one decimal point. Could just round it here, see suggestion below. Or set a useable field in `OnInitialize`.
```suggestion
	<i class="fa stars stars-child rating-uncolored" style="width: @(100 - Math.Round(ReviewAverage, 1) * 100 / 5)%"></i>
```

Reply:
1. From where do you draw the conclusion that only one decimal point can be handled
2. Even if that's the case, why bother manually rounding it when it is presumably disregarded later anyways?

Reply:
Well, I tested it and it didn't work as soon as the calculated value is not divisible by 5. Which is not the same, but rounding solves that issue.